Fix for issue with duplicated block on Magento 2.3 and problem with sorting tab on PDP
Issue reported on HaarShop.
Error log: "main.CRITICAL: An element with a "amfile.attachment" ID already exists."

diff --git a/etc/di.xml b/etc/di.xml
index 2f37353..2ee3c92 100644
--- a/etc/di.xml
+++ b/etc/di.xml
@@ -69,7 +69,7 @@
     </type>
 
     <type name="Magento\Catalog\Block\Product\View\Description">
-        <plugin name="Amasty_ProductAttachment::ProductViewTabPosition" type="Amasty\ProductAttachment\Plugin\TabPosition" />
+        <plugin name="Amasty_ProductAttachment::ProductViewTabPosition" type="Amasty\ProductAttachment\Plugin\TabPosition" disabled="true"/>
     </type>
 
     <type name="Amasty\ProductAttachment\Block\Product\AttachmentsTab">
